recipe/build.sh

```diff
@@ -11,14 +11,17 @@ pushd src
 # Install node-gyp globally as a fix for NodeJS 18.18.2 https://github.com/microsoft/vscode/issues/194665
 npm i -g node-gyp
 
-# fetch the version of ripgrep, removes a ^ if present
-VSCODE_RIPGREP_VERSION=$(jq -r '.dependencies."@vscode/ripgrep" | sub("^\\^"; "")' package.json)
-# ripgrep tries to download binary from github, copying the ripgrep executable to the cache location so that it doesn't do that 
-mkdir /tmp/vscode-ripgrep-cache-${VSCODE_RIPGREP_VERSION}
-cp "${PREFIX}/bin/rg" /tmp/vscode-ripgrep-cache-${VSCODE_RIPGREP_VERSION}/
+# Install all dependencies except @vscode/ripgrep
+VSCODE_RIPGREP_VERSION=$(jq -r '.dependencies."@vscode/ripgrep"' package.json)
+mv package.json package.json.orig
+jq 'del(.dependencies."@vscode/ripgrep")' package.json.orig > package.json
 
 yarn install
```

Comment:
Should we have error handling if `yarn install` fails?

Reply:
`yarn install` is a critical step here - so if that fails the recipe build would also fail and log the error. We would then need to fix the recipe.